app/(nms)/subscribers/modals.tsx

```diff
@@ -451,45 +453,218 @@ export function EditSubscriberModal({ subscriber, token, closeFn }: editSubscrib
+                <Col size={2}>
+                  <div className="u-align--right">
+                   <Button
+                      appearance="positive"
+                      type="button"
+                      onClick={() => {navigator.clipboard.writeText(subscriberValues.plmnId + subscriberValues.msin)}}
+                    >
+                      Copy
+                    </Button>
+                  </div>
+                </Col>
+              </Row>
+            </Col>
+          </Row>
+        </fieldset>
+        <fieldset><legend>Authentication</legend>
+          <Row>
+            <Col size={3}>OPC</Col>
+            <Col size={9}>
+              <Row className="p-form__control">
+                <Col size={7}>
+                  <div>{subscriberValues.opc}</div>
+                </Col>
+                <Col size={2}>
+                  <div className="u-align--right">
+                   <Button
+                      appearance="positive"
+                      type="button"
+                      onClick={() => {navigator.clipboard.writeText(subscriberValues.opc)}}
+                    >
+                      Copy
+                    </Button>
+                  </div>
+                </Col>
+              </Row>
+            </Col>
+          </Row>
+          <Row>
+            <Col size={3}>Key</Col>
+            <Col size={9}>
+              <Row className="p-form__control">
+                <Col size={7}>
+                  <div>{subscriberValues.key}</div>
+                </Col>
+                <Col size={2}>
+                  <div className="u-align--right">
+                    <Button
+                      appearance="positive"
+                      type="button"
+                      onClick={() => {navigator.clipboard.writeText(subscriberValues.key)}}
+                    >
+                      Copy
+                    </Button>
+                  </div>
+                </Col>
+              </Row>
+            </Col>
+          </Row>
+          <Row>
+            <Col size={3}>Sequence Number</Col>
+            <Col size={9}>
+              <Row className="p-form__control">
+                <Col size={7}>
+                  <div>{subscriberValues.sequenceNumber}</div>
+                </Col>
+                <Col size={2}>
+                  <div className="u-align--right">
+                    <Button
+                      appearance="positive"
+                      type="button"
+                      onClick={() => {navigator.clipboard.writeText(subscriberValues.sequenceNumber)}}
+                    >
+                      Copy
+                    </Button>
+                  </div>
+                </Col>
+              </Row>
+            </Col>
+          </Row>
+        </fieldset>
+        <fieldset><legend>pySim command</legend>
+          <Row className="p-form__control">
+            <Col size={10}>
+              <CodeSnippet style={{ opacity: 0.33, cursor: "not-allowed" }}
```

Comment:
```suggestion
              <CodeSnippet
```
I think that all the fields should either be black or all of them gray.  We should be able to select each of them or none of them